examples/00-Landscaper-Configuration.yaml

```diff
@@ -8,4 +8,31 @@ repositoryContext: # define the default repository context for installations
 registries:
   oci:
     configFiles:
-     - "/home/some/path/docker.json"
\ No newline at end of file
+     - "/home/some/path/docker.json"
+
+crdManagement:
+  deployCrd: true
+  forceUpdate: true
+
+deployerManagement:
+  #  disable: false
+  namespace: "ls-system"
+  agent:
+    apiVersion: config.landscaper.gardener.cloud/v1alpha1
+    kind: AgentConfiguration
+    disable: false # is automatically deactivated of the deployer management is disabled.
```

Comment:
I think we use that "inverse" logic in a lot of places (btw also gardener and k8s using it) simply because go defaults booleans to `false`. 
Otherwise every bool that is defaulted has to be a pointer (otherwise the difference between `defaulted by go` and `not set` is not clear).
A bool pointer would make the actual code usage more complex..

```
type mystrcut struct {
  Enabled *bool
  Disabled bool
}
```

But in general I'm with you and I think that such language specifics should not influence any external api.

I suggest that we leave it for the alpha version and may update all cases when we move to the beta version.
For now I want to keep the api consistent but I have a idea how we could fix both issues in the next version.

@Diaphteiros can you open a ticket for that